src/chunk/render.rs

```diff
@@ -45,57 +45,126 @@ macro_rules! build_chunk_pipeline {
                 ..PipelineDescriptor::new(ShaderStages {
                     vertex: shaders
                         .add(Shader::from_glsl(ShaderStage::Vertex, include_str!($file))),
-                    fragment: Some(shaders.add(Shader::from_glsl(
-                        ShaderStage::Fragment,
-                        include_str!("tilemap.frag"),
-                    ))),
+                    fragment: Some(shaders.add(Shader::from_glsl(ShaderStage::Fragment, {
+                        let _wasm = false;
+                        #[cfg(target_arch = "wasm32")]
+                        let _wasm = true;
+
+                        if _wasm {
+                            include_str!("render_web/tilemap.frag")
+                        } else {
+                            include_str!("render_native/tilemap.frag")
+                        }
+                    }))),
                 })
             }
         }
     };
 }
 
+#[cfg(not(target_arch = "wasm32"))]
 build_chunk_pipeline!(
     CHUNK_SQUARE_PIPELINE,
     2110840099625352487,
     build_chunk_square_pipeline,
-    "tilemap-square.vert"
+    "render_native/tilemap-square.vert"
```

Comment:
Since these are the "intended" target, think that I might keep these in the root folder then web in another.